build/int.cloudbuild.yaml

```diff
@@ -38,4 +38,4 @@ tags:
 - 'integration'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.12.0'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.13.11'
```

Comment:
Was this intended to now test with TF 0.13? Usually we have just been relaxing the constraint to allow usage with 0.13, but not actively testing yet. IMO we could move testing to 0.13, however if we do introduce 0.13 features, we should call that out in the [readme](https://github.com/terraform-google-modules/terraform-google-bastion-host#software) and make a breaking release.

Reply:
I think this is more of an issue of how we test. If modules should be able to support 12 and 13, we should test with both IMO, I think this would be a simple matter of ensuring both versions exist in the container, then doing a `terraform validate` with each command. Thoughts?

Reply:
@onetwopunch officially we dont _support_  13 yet and hence why we only relax constraints for end users who wish to try it out. I think either is fine as long as we define what version we are testing with and interim testing both is also good for transition. 

For a full 0.13 test suite, we should have at least one test that makes use of module `for_each` and `depends_on`. For instance GKE module works with 0.13 but breaks with module `for_each`. 
/cc @morgante

Reply:
Ahh good to know, I'll relax the testing requirements til the rest of CFT officially supports it.